can_msgs/CMakeLists.txt

```diff
@@ -2,22 +2,53 @@ cmake_minimum_required(VERSION 3.5)
 project(can_msgs)
 
 find_package(carma_cmake_common REQUIRED)
```

Comment:
Looks like this was removed from package xml, but not from here

Reply:
This wasn't removed its still there. That CI error is missing the ros_environment package. Which I've recently realized is caused by the extra catkin install. I'm working on a fix for it still